packit_service/worker/copr_build.py

```diff
@@ -157,11 +157,12 @@ def run_copr_build(self) -> HandlerResults:
             return HandlerResults(success=False, details={"msg": msg})
 
         except Exception as ex:
-            msg = f"There was an error while running a copr build. {msg_retrigger}"
-            self.project.pr_comment(self.event.pr_id, msg)
-            logger.error(f"error while running a copr build: {ex}")
-            msg = f"There was an error while running the build: {ex}"
-            r.report("failure", msg, check_name=check_name)
+            msg = f"There was an error while running a copr build: {ex}"
+            logger.error(msg)
+            self.project.pr_comment(self.event.pr_id, f"{msg}\n{msg_retrigger}")
+            r.report(
+                "failure", "Build failed, check latest comment.", check_name=check_name
```

Comment:
Huge nit:
`Build failed, check latest comment.` -> `Build failed, check the latest comment for more details.` or is it too long?

Reply:
It seems to be longer than what can be displayed.
What about a compromise? (check now)

Reply:
+1